sdcm/sct_config.py

```diff
@@ -1716,11 +1749,6 @@ def _load_environment_variables(self):
 
         return environment_vars
 
-    def _update_environment_variables(self, replace=False):
```

Comment:
why are you remove this ? isn't there k8s related code building upon this  ?

Reply:
It was added by Evgeniy when yaml files were using SCT's env vars in the beginning of the K8S support.
Now, we do not use env vars directly. Only prepared ones beforehand.
So, it is not only useless now, but makes the new approach for the config options fail because it create a loop of changed vars.

Reply:
o.k.